lib/wda/xcodebuild.js

```diff
@@ -145,7 +154,10 @@ class XcodeBuild {
   }
 
   async cleanProject () {
-    for (const scheme of [LIB_SCHEME, RUNNER_SCHEME]) {
+    const libScheme = _.toLower(this.platformName) === _.toLower(PLATFORM_NAME_TVOS) ? LIB_SCHEME_TV : LIB_SCHEME_IOS;
```

Comment:
Perhaps this equality check could be in a helper function (e.g., `isTvOS`) since it happens again and again, which might be able to be memoized.

Alternatively, it could be in a function like `getSchemes`
```js
getSchemes () {
  const isTvOS = _.toLower(this.platformName) === _.toLower(PLATFORM_NAME_TVOS);
  const libScheme = isTvOS ? LIB_SCHEME_TV : LIB_SCHEME_IOS;
  const runnerScheme = isTvOS ? RUNNER_SCHEME_TV : RUNNER_SCHEME_IOS;
  return {
    libScheme,
    runnerScheme,
  };
}
```